core/cleaner/src/main/java/com/cognifide/aet/cleaner/configuration/CleanerSchedulerConf.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.cleaner.configuration;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "AET Cleaning Scheduler Service", description = "AET Cleaning Scheduler Service")
+public @interface CleanerSchedulerConf {
+
+  String COMPANY_NAME = "companyName";
+
+  String PROJECT_NAME = "projectName";
+
+  String REMOVE_OLDER_THAN = "removeOlderThan";
+
+  String KEEP_N_VERSIONS = "keepNVersions";
+
+  String SCHEDULE_CRON = "schedule";
+
+  String DRY_RUN = "dryRun";
```

Comment:
All above constants should be changed to `label`s which were used in old annotations